### PR TITLE
Tamar fix

### PR DESCRIPTION
**Pull request by GrishaDev**
it fixes
https://github.com/rabiran/Kartoffel/issues/149

### CODE REVIEW

File: src/person/person.spec.ts [hunk 1]
```diff
@@ -577,42 +577,42 @@ describe('Persons', () => {
     });
     it('Should return the updated person', async () => {
       const person = await Person.createPerson(<IPerson>{ ...personExamples[0] });
+      const updateObject: any = {};
+      updateObject.job = 'Programmer';
+      updateObject.rank = RANK[0];
+      updateObject.responsibility = RESPONSIBILITY[1];
+      updateObject.responsibilityLocation = new Types.ObjectId(dbIdExample[0]);
+      updateObject.serviceType = SERVICE_TYPE[7];
```
Comment (eladex) [inline src/person/person.spec.ts:585]: maybe just 
`const updateObject: Partial<IPerson> =  {
  job: ...,
  rank: ...,
  ...more fields
}
`

**Comment by eladex** [comment]
Did all the tests pass?

**Comment by GrishaDev** [comment]
Ofcourse, also we dont have something that runs tests on pull requests?

**Comment by eladex** [comment]
Unfortunately  no,
I don't know what devops are doing, CI could be nice

**Pull request merged**